src/OrchardCore.Modules/Orchard.Users/Services/SetupEventHandler.cs

```diff
@@ -30,15 +25,7 @@ public SetupEventHandler(IServiceProvider serviceProvider)
         public Task Setup(string userName, string email, string password, Action<string, string> reportError)
         {
             var userService = _serviceProvider.GetRequiredService<IUserService>();
-
-            var superUser = new User
-            {
-                UserName = userName,
-                Email = email,
-                RoleNames = { "Administrator" }
-            };
-
-            return userService.CreateUserAsync(superUser,password,reportError);
+            return userService.CreateUserAsync(userName, email, new string [] { "Administrator" }, password, reportError);
```

Comment:
The `Administrator` role is already added in the recipe. It should not be hard-coded here.

Reply:
But this is the creation of a superuser. This is not a role creation. Am I missing something?